src/components/ToDoList.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import '../styles/todo.scss';
+import '../styles/todoapp.scss';
+import { StateContext } from './ToDoContext';
+import { FilterButtons } from '../types/FilterType';
+import { ToDoItem } from './ToDoItem';
+
+export const ToDoList: React.FC = () => {
+  const { filterButton } = useContext(StateContext);
```

Comment:
This one should have name `activeFilter` or something like that. Current name don't tell me much about this variable.